01-js-intro-and-jquery/app.js

```diff
@@ -0,0 +1,27 @@
+var main = function() {
+    
+    $('.hidden').fadeIn(5000, function() {
+
+        $( '.alias' ).focus()
+    });
+
+    $('#btn').click( function( event ) {
+
+        $.get("http://bootcamp.aws.af.cm/welcome/yourname", function(text) {
```

Comment:
**Minor:** use double quotes only for JSON notation on .json files (as a coding convention we use single quotes within JS files).